app/templates/components/preprint-form-authors.hbs

```diff
@@ -1,56 +1,142 @@
-        <label>{{radio-button name="bibliographic" value=false checked=newContributorIsBibliographic}} Non-Bibliographic </label>
-    </div>
-    <div class="form-group">
-        <button {{action 'addContributor' newContributorId newContributorPermissions newContributorIsBibliographic}} class="btn btn-primary">Add</button>
+    <div class="col-xs-12 col-sm-6">
+        <div class="author-header">
+            <h2 class="header-inline"> Authors </h2>
+            <span>
+                <i class="fa fa-question-circle permission-info"
+                    data-toggle="popover"
+                    data-title="Author Information"
+                    data-trigger="hover"
+                    data-html="true"
+                    id="author-popover">
+                </i>
+            </span>
+        </div>
+        <p class="drag-drop small"> Drag and drop authors to change listing order. </p>
+        <div class="panel-body">
+            <table class="table author-table">
+                <tr>
+                    <th> </th>
+                    <th> Name </th>
+                    <th>
+                        Permissions
+                        <span>
+                            <i class="fa fa-question-circle permission-info"
+                                data-toggle="popover"
+                                data-title="Permission Information"
+                                data-trigger="hover"
+                                data-html="true"
+                                data-placement="bottom"
+                                id="permissions-popover">
+                            </i>
+                        </span>
+                    </th>
+                    <th class="bib-padding">
+                        In Citation
+                        <span>
+                            <i class="fa fa-question-circle permission-info"
+                                data-toggle="popover"
+                                data-title="Citation Information"
+                                data-trigger="hover"
+                                data-html="true"
+                                data-placement="bottom"
+                                id="bibliographic-popover">
+                            </i>
+                        </span>
+                    </th>
+                    <th> </th>
+                </tr>
+                {{#each contributors as |contrib|}}
+                    <tr class="contributor-row">
+                        <td> <span class="fa fa-bars sortable-bars"></span> </td>
+                        <td>
+                            <img class="m-l-xs" src={{contrib.users.links.profile_image}} height=40 width=40>
+                            {{#if contrib.unregisteredContributor}}
+                                {{contrib.unregisteredContributor}}
+                            {{else}}
+                                <a href={{contrib.users.links.html}}> {{contrib.users.fullName}} </a>
```

Comment:
Line 107 and here: use helper methods as possible. Accessing to model fields should not depend on the internal structure of the API JSON response; these helper methods provide a consistent and abstracted way to get that value.

 https://github.com/abought/ember-osf/blob/ea1462dfd277155da6a3d803075fd7e375bb8638/addon/models/user.js#L37-L37